ts/indicators/volume-by-price.src.ts

```diff
@@ -361,11 +361,18 @@ seriesType<Highcharts.VBPIndicator>(
             init: boolean
         ): void {
             var series = this,
-                attr: Highcharts.SVGAttributes = {};
-
-            if (!init) {
-                attr.translateX = series.yAxis.pos;
-                (series.group as any).animate(
+                inverted = series.chart.inverted,
+                group = series.group as Highcharts.SVGElement,
+                attr: Highcharts.SVGAttributes = {},
+                translate,
+                position;
+
+            if (!init && group) {
```

Comment:
Now that we check the group, we can also remove the cast:

`group = series.group as Highcharts.SVGElement,` 
=>
`group = series.group,`

Reply:
Ahh, yes. Corrected, thanks!